CommonTools/PileupAlgos/plugins/PuppiProducer.cc

```diff
@@ -351,6 +356,7 @@ void PuppiProducer::fillDescriptions(edm::ConfigurationDescriptions& description
   desc.add<bool>("UseDeltaZCut", true);
   desc.add<double>("DeltaZCut", 0.3);
   desc.add<double>("PtMaxCharged", 0.);
+  desc.add<double>("EtaMaxCharged", 0.);
```

Comment:
I see some potential (even if perhaps unlikely) danger due to rounding approximations in setting these parameters at "0", when there are checks like
```C++
   if ((fPtMaxCharged > 0) ...
```

Wouldn't be identical, for the point of view of the results, but safer, setting the value to "-1" (e.g) instead of "0" for the following parameters?
- PtMaxCharged
- EtaMaxCharged
- PtMaxPhotons